src/components/Footer/Footer.jsx

```diff
@@ -26,88 +30,96 @@ const styles = (theme) => ({
-    // whiteSpace: 'nowrap',
     color: 'inherit',
+    fontWeight: 600,
+    textDecoration: 'none',
   },
-  link: {
-    float: 'right',
-    fontSize: 14,
-    '& a': {
-      color: 'inherit',
-      marginRight: 20,
-      whiteSpace: 'pre-wrap',
-    },
-    '& a:last-of-type': {
-      marginRight: 0,
-    },
-  },
-  copyright: {
-    '&::before': {
-      content: '""',
-      marginRight: 10,
-      marginLeft: 10,
-      borderLeft: '1px solid white',
-      whiteSpace: 'pre-wrap',
-    },
+  buttonLink: {
+    color: 'inherit',
+    fontWeight: 400,
   },
 });
 
-const Footer = ({ classes }) => (
-  <div className={classes.root}>
-    <div className={classes.info}>
-      <Typography variant="subtitle2">
-        The Lavender Book is a crowd-sourced application created
-        by the team at &nbsp;
-        <a href="https://nbjc.org/" target="_blank" rel="noreferrer" className={classes.externalLink}>
-          National Black Justice Coalition
-        </a>
-        &nbsp; and &nbsp;
-        <a href="https://outintech.com/" target="_blank" rel="noreferrer" className={classes.externalLink}>
-          Out in Tech
-        </a>
-      </Typography>
-    </div>
-    <div className={classes.links}>
-      <Typography className={classes.link}>
-        <a size="small" href="/community-guidelines">
-          Community Guidelines
-        </a>
-        <a size="small" href="/terms-of-service">
-          Terms of Service
-        </a>
-        <a size="small" href="https://nbjc.org/privacy-policy/" rel="noreferrer" target="_blank">
-          Privacy Policy
-        </a>
-        <a size="small" href="/infringement-policies">
-          Infringement Policies
-        </a>
-        <Box component="span" className={classes.copyright}>
-          Copyright 2021 NBJC
-        </Box>
-      </Typography>
+const Footer = ({ classes }) => {
+  const history = useHistory();
+
+  return (
+    <div
+      className={classes.root}
+    >
+      <Grid container className={classes.info} style={{ display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+        <Grid item xs={12} md={5}>
+          <Typography variant="subtitle2">
+            The Lavender Book is a crowd-sourced application created
+            by the team at &nbsp;
+            <a href="https://nbjc.org/" target="_blank" rel="noreferrer" className={classes.externalLink}>
+              National Black Justice Coalition
+            </a>
+            &nbsp; and &nbsp;
+            <a href="https://outintech.com/" target="_blank" rel="noreferrer" className={classes.externalLink}>
+              Out in Tech
+            </a>
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Button
+            className={classes.buttonLink}
+          >
+            Community Guidelines
+          </Button>
+        </Grid>
+        <Grid item>
+          <Button
+            className={classes.buttonLink}
+            onClick={() => {
+              history.push({
```

Comment:
Why do we need this instead of just a `href` ?

Reply:
Href elements would trigger a page refresh which would reset the application states. In this instance it means that the application would forget if the user was logged in or not which is relevant for other components and single page applications. 

If not useHistory, we can also use NavLink or Link which is part of React Router.